core/modules/admin_bar/admin_bar.inc

```diff
@@ -490,11 +490,9 @@ function admin_bar_links_menu($tree) {
     // Remove description to prevent mouseover tooltip clashes.
     unset($data['link']['localized_options']['attributes']['title']);
 
-    // Make action links (typically "Add ...") appear first in dropdowns.
-    // They might appear first already, but only as long as there is no link
-    // that comes alphabetically first (e.g., a node type with label "Ad").
+    // Make action links (typically "Add ...") appear last in dropdowns.
```

Comment:
@jenlampton I believe that we are addressing this one separately in https://github.com/backdrop/backdrop-issues/issues/4586, so this change here needs to be reverted please.